### PR TITLE
MGDOBR-135 Automate the demo of creation of a managed service connector for slack using the Managed Connector

### PR DESCRIPTION
**Pull request by lucamolteni**
Simple console application to create a Slack Managed Connector 

[MGDOBR-135](https://issues.redhat.com/browse/MGDOBR-135) 

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: use-cases/managed-connectors-integration/pom.xml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-arc</artifactId>
+    </dependency>
+
+
+    <dependency>
+      <groupId>com.redhat.cloud</groupId>
+      <artifactId>connector-management-sdk</artifactId>
+      <version>0.0.3</version>
```
Comment (danielezonca) [inline use-cases/managed-connectors-integration/pom.xml:66]: What about move to property/dependencyManagement?

File: use-cases/managed-connectors-integration/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Managed Connectors Integration
```
Comment (danielezonca) [inline use-cases/managed-connectors-integration/README.md:1]: Can you please add a prerequisite section with link to required tools? As far as I see it requires rhoas and kcat CLI

**Comment by lucamolteni** [comment]
Connector SDK has a problem with the DTO definition, it needs to be compiled locally. We can't merge it before it's fixed

**Comment by lucamolteni** [comment]
> Connector SDK has a problem with the DTO definition, it needs to be compiled locally. We can't merge it before it's fixed

It should be fixed here

**Review by robpblake — APPROVED** [review]
Thanks @lucamolteni !

**Review by danielezonca — APPROVED** [review]

**Review by kostola — APPROVED** [review]

**Pull request merged**